core/domain/collection_domain.py

```diff
@@ -401,12 +404,12 @@ def validate_skill_id(skill_id):
                 'Expected skill ID to have length at least 6, received %s' %
                 skill_id)
 
-        if skill_id[0:5] != 'skill':
+        if skill_id[:5] != _SKILL_ID_PREFIX:
             raise utils.ValidationError(
                 'Expected skill ID to begin with \'skill\', received %s' %
```

Comment:
Validation error string should probably be updated to interpolate _SKILL_ID_PREFIX.

Reply:
Oops, done, thanks!